libs/auth-nest-tools/src/lib/jwt.strategy.spec.ts

```diff
@@ -106,7 +106,7 @@ describe('JwtStrategy#validate', () => {
     expect(user.nationalId).toEqual(payload.nationalId)
     expect(user.scope).toEqual(payload.scope)
     expect(user.client).toEqual(payload.client_id)
-    expect(user.authorization).toEqual(request.headers.authorization)
+    expect(user.authorization).toEqual('')
```

Comment:
This should be able to set the correct authorization. This was working before your change as the header was just copied as is to the `authorization` property. But with the extractor it only finds the token value if the `Authorization` header is `Bearer token`. We should update L95 to be `authorization: 'Bearer token'` and then keep this assert as is.

It would also be nice to add a test case that adds the token to the request body and provides the correct `auth.authorization` value.

Reply:
Yes, makes sense.